### ISSUE TITLE
source_licenses fails for maven:ch.qos.logback:logback-access:1.1.9

### ISSUE THREAD

**Issue by fridex** [fabric8-analytics/fabric8-analytics-worker#146]
```
{"task_name": "source_licenses", "node_args": {"recursive_limit": 0, "version": "1.1.9", "document_id": 1116620, "ecosystem": "maven", "force_graph_sync": false, "force": true, "_release": "maven:ch.qos.logback:logback-access:1.1.9", "_audit": {"version": "v1", "started_at": "2017-07-18T07:11:24.117505", "ended_at": "2017-07-18T07:11:24.296110"}, "name": "ch.qos.logback:logback-access"}, "flow_name": "bayesianAnalysisFlow", "event": "TASK_START", "task_id": "f9e8f9f6-8276-4016-95f9-90620a5170fb", "queue": "prod_ingestion_LicenseCheckTask_v0", "parent": {}, "dispatcher_id": "e89a6924-8128-4bf3-ad18-4a6618678e3c"}
/opt/scancode-toolkit/scancode: line 114: 71834 Killed                  $SCANCODE_ROOT_DIR/bin/scancode "$@"
2017-07-18 08:30:58,978 [WARNING] f8a_worker.utils: command ['/opt/scancode-toolkit/scancode', '--license', '--license-score', '20', '--only-findings', '--processes', '1', '--quiet', '--strip-root', '--timeout', '120', '/var/lib/f8a_worker/worker_data/maven/ch.qos.logback:logback-access/1.1.9/extracted_jar', '--ignore', '"*.pyc"', '--ignore', '"*.so"', '--ignore', '"*.dll"'] ended with 137

2017-07-18 08:30:58,980 [ERROR] f8a_worker.utils: exception is fatal
2017-07-18 08:30:58,980 [ERROR] LicenseCheckTask: License scan failed
Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/f8a_worker/utils.py", line 255, in get_command_output
    out = check_output(args, universal_newlines=True, **kwargs)
  File "/usr/lib64/python3.4/subprocess.py", line 617, in check_output
    raise CalledProcessError(retcode, process.args, output=output)
subprocess.CalledProcessError: Command '['/opt/scancode-toolkit/scancode', '--license', '--license-score', '20', '--only-findings', '--processes', '1', '--quiet', '--strip-root', '--timeout', '120', '/var/lib/f8a_worker/worker_data/maven/ch.qos.logback:logback-access/1.1.9/extracted_jar', '--ignore', '"*.pyc"', '--ignore', '"*.so"', '--ignore', '"*.dll"']' returned non-zero exit status 137

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/lib/python3.4/site-packages/f8a_worker/workers/license.py", line 96, in execute
    timeout=1200)
  File "/usr/lib/python3.4/site-packages/f8a_worker/utils.py", line 239, in get_command_output
    return get_command_output(args, graceful, is_json, **kwargs)
  File "/usr/lib/python3.4/site-packages/f8a_worker/utils.py", line 265, in get_command_output
    raise TaskError("Error during running command %s: %r" % (args, ex.output))
f8a_worker.errors.TaskError: Error during running command ['/opt/scancode-toolkit/scancode', '--license', '--license-score', '20', '--only-findings', '--processes', '1', '--quiet', '--strip-root', '--timeout', '120', '/var/lib/f8a_worker/worker_data/maven/ch.qos.logback:logback-access/1.1.9/extracted_jar', '--ignore', '"*.pyc"', '--ignore', '"*.so"', '--ignore', '"*.dll"']: ''
```

**Comment by jpopelka** [comment]
The `exit status 137` most likely means that the scancode process was killed.

This should already be fixed by #128 but the fix had probably not been deployed yet at the time when you reported this.

**Comment by jpopelka** [comment]
Duplicate of #127
